common/src/main/java/com/wynntils/models/players/FriendsModel.java

```diff
@@ -180,22 +231,40 @@ private void resetData() {
     public void requestData() {
         if (McUtils.player() == null) return;
 
-        if (System.currentTimeMillis() - lastFriendRequest < 250) {
-            WynntilsMod.info("Skipping friend list request because it was requested less than 250ms ago.");
-            return;
+        if (System.currentTimeMillis() - lastFriendRequest > REQUEST_RATELIMIT) {
```

Comment:
I think we have a class for exactly that use (called `ThrottledSupplyer`)

Reply:
I don't really think it is worth it to be used here. The current system works by sending a command and setting a variable, then listening for a response in a separate event. We can't really easily call .get() on the chat event without doing some pretty heavy refactoring to this class. If someone still believes this change should happen lmk.